src/nvim/ex_eval_defs.h

```diff
@@ -6,7 +6,7 @@
 /// There is no CSF_IF, the lack of CSF_WHILE, CSF_FOR and CSF_TRY means ":if"
 /// was used.
 enum {
-  CSF_TRUE     = 0x0001,  ///< condition was TRUE
+  CSF_TRUE     = 0x0001,  ///< condition was true
```

Comment:
keep

Reply:
```
						*TRUE* *FALSE* *Boolean*
For boolean operators Numbers are used.  Zero is FALSE, non-zero is TRUE.
You can also use |v:false| and |v:true|, in Vim9 script |false| and |true|.
When TRUE is returned from a function it is the Number one, FALSE is the
number zero.

```
```
						*false* *true* *null* *E1034*
In Vim9 script one can use the following predefined values: >
	true
	false
	null
	null_blob
	null_channel
	null_dict
	null_function
	null_job
	null_list
	null_partial
	null_string
`true` is the same as `v:true`, `false` the same as `v:false`, `null` the same
as `v:null`.
```

Reply:
so basically when referring to the _Vimscript_ Booleans in the C code, we should still use `TRUE` and `FALSE` to align with the Vimdocs?

Reply:
And `TRUE` may also refer to `CSF_TRUE` macro, which is upper-case.